src/components/table/Table.vue

```diff
@@ -63,7 +63,8 @@
                                 'is-sortable': column.sortable
                             }"
                             :style="{
-                                width: column.width === undefined ? null : column.width + 'px'
+                                width: column.width === undefined ? null :
+                                (column.width == 'number' ? column.width + 'px' : column.width )
```

Comment:
This is not really how you verify that something is a number

Reply:
You are right, sorry it is really `typeof column.width == 'number'`